src/containers/WalletDetails/index.js

```diff
@@ -99,6 +70,23 @@ export class WalletDetails extends React.PureComponent {
             key={`${match.url}/overview`}
           >
           </TabPane>
+        );
+      }
+      if (feature === 'buy_eth') {
```

Comment:
Looks like an `if...else if` or a `switch... case` would be more appropriate instead of multiple "ifs"??

Reply:
Agreed, I use switch...case now